deploy/cluster_role_binding_pv.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: local-operator-pv-binding
+subjects:
+  - kind: ServiceAccount
+    name: local-storage-operator
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: 'system:persistent-volume-provisioner'
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
These are just file renames? why?

Reply:
These are the yamls generated by the operator-sdk. operator-sdk wants these yamls in these formats otherwise it gives parsing error when you try to add a new API.

Reply:
same as above.